diff --git a/src/session_record.js b/src/session_record.js
index 7626a3927fadcdf7f66b3d0c9cf8c63d33da579b..6849c1167bcd2ea730c2258fcf692313ff3ed5d7 100644
--- a/src/session_record.js
+++ b/src/session_record.js
@@ -267,7 +267,5 @@ class SessionRecord {
     closeSession(session) {
         if (this.isClosed(session)) {
-            console.warn("Session already closed", session);
             return;
         }
-        console.info("Closing session:", session);
         session.indexInfo.closed = Date.now();
     }
@@ -278,5 +276,4 @@ class SessionRecord {
     openSession(session) {
         if (!this.isClosed(session)) {
             console.warn("Session already open");
         }
-        console.info("Opening session:", session);
         session.indexInfo.closed = -1;
     }
@@ -298,7 +295,6 @@ class SessionRecord {
                 }
             }
             if (oldestKey) {
-                console.info("Removing old closed session:", oldestSession);
                 delete this.sessions[oldestKey];
             } else {
                 throw new Error('Corrupt sessions object');
